site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
+
+Chrome 91 adds the following features to the extensions platform.
+
+#### ES modules for service workers
+
+JavaScript supports modules in service workers. Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts. (https://www.chromestatus.com/guide/edit/4609574738853888)
+
+#### chrome.action.getUserSettings() available
+
+The new chrome.action.getUserSettings() API allows extensions to determine if the user has pinned the extension to the main toolbar.
```

Comment:
put this method call in backticks?

Reply:
and also link to it?

https://developer.chrome.com/docs/extensions/reference/action/#method-getUserSettings

Reply:
```suggestion
The new [`chrome.action.getUserSettings()`][get-user-settings] API allows extensions to determine if the user has pinned the extension to the main toolbar.

[get-user-settings]: https://developer.chrome.com/docs/extensions/reference/action/#method-getUserSettings
```